### ISSUE TITLE
TypeError: in method 'delete_Font', argument 1 of type 'struct Font *' (version 1.19.4)

### ISSUE THREAD

**Issue by Augus1999** [pymupdf/PyMuPDF#1521]
## Describe the bug
Before version 1.19.4, when trying to open a font file that pymupdf doesn't support it raises a RuntimeError and a mupdf warning `mupdf: FT_New_Memory_Face((null)): unknown file format`. However, at version 1.19.4 the exception becomes unable to be tracked by python: 
```
Exception ignored in: <function Font.__del__ at 0x000002D06A2AAB80>
Traceback (most recent call last):
  File "\.venv\lib\site-packages\fitz\fitz.py", line 9063, in __del__
    self.__swig_destroy__(self)
TypeError: in method 'delete_Font', argument 1 of type 'struct Font *'
```

## To Reproduce
The error occurs when running the following code:
```python
def find_font(font_dirs: list) -> (dict, dict):
    """
    find all TrueType font files (.ttf): all their font name and file addresses
    then write their directories to a json file

    :param font_dirs: the directions where font files locate
    :return: two dictionaries => {font name: font file address} &
                                 {font file address: font name}
    """
    name_dict = {}
    dir_dict = {}
    for font_dir in font_dirs:
        for file_name in os.listdir(font_dir):
            full_name = os.path.join(
                font_dir,
                file_name,
            )
            try:
                font_name = fitz.Font(fontfile=full_name).name
                name_dict[font_name] = str(Path(full_name))
                dir_dict[str(Path(full_name))] = font_name
            except RuntimeError:
                pass
            except TypeError:
                pass
    return name_dict, dir_dict

font_paths = QtCore.QStandardPaths.standardLocations(
                QtCore.QStandardPaths.FontsLocation,
            )
name_dict, file_dict = find_font(font_paths)
```

## Expected behavior
As usual it should raise a RuntimeError.

## Your configuration (mandatory)
 - Windows 11
 - Python 3.9.7, 64 bit
 - PyMuPDF 1.19.4, installed via `pip install pymupdf`

**Comment by JorjMcKie** [comment]
This is a duplicate of #1510. Please drop me note if you need a pre-version wheel.

**Comment by Augus1999** [comment]
Hi, thanks for reply. I'm lucky enough that this error doesn't terminate my application.🪶

**Comment by JorjMcKie** [comment]
Fixed in v1.19.5 currently being uploaded.
